### PR TITLE
Fix openj9 javadoc

### PR DESCRIPTION
**Pull request by lumpfish**
This PR is the equivalent of https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/212 for the openj9-openjdk-jdk (jdk head) repository.

### CODE REVIEW

File: closed/make/Docs.gmk [hunk 1]
```diff
@@ -21,3 +21,103 @@
...
+        $(OPENJ9_SUBST_COMMAND) < "$(OPENJ9_JAVADOC_TEMP)" > "$(DOCS_OUTPUTDIR)/api/$$path" ; \
+        $(RM) "$(OPENJ9_JAVADOC_TEMP)" ; \
+      fi ; \
+    done \
+    #
+
+# Dependency on the openjdk marker file ensures the footers are only
+# replaced after the javadoc has been created.
+# on some platforms. 
+$(OPENJ9_JAVADOC_MARKER) : $(OPENJDK_JAVADOC_MARKER)
```
Comment (keithc-ca) [inline closed/make/Docs.gmk]: This is different than jdk11: does this make sense to apply there?

nit: Please remove trailing whitespace from all lines.

File: closed/make/common/Modules.gmk [hunk 2]
```diff
@@ -29,6 +29,9 @@ MODULES_FILTER += \
     jdk.internal.vm.ci \
     jdk.internal.vm.compiler \
     jdk.internal.vm.compiler.management \
+    jdk.jfr \
```
Comment (keithc-ca) [inline closed/make/common/Modules.gmk]: The copyright date needs to be updated in this file.

Comment (lumpfish) [reply closed/make/common/Modules.gmk]: Updated.  Thanks for spotting.

File: closed/make/Docs.gmk (hunk 1 above)
Comment (lumpfish) [reply closed/make/Docs.gmk]: I found 2 trailing spaces.  Did you see others?
I also removed the 'on some platforms' line which shouldn't be there.

**Comment by keithc-ca** [comment]
This looks good. Can you please squash to a single commit?
We'll also want an equivalent change for jdk13.

**Comment by lumpfish** [comment]
Squashed.  I'll create a PR for jdk13.

**Comment by keithc-ca** [comment]
Jenkins compile aix,osx,plinux,xlinux,zlinux,win jdknext

**Pull request merged**